### ISSUE TITLE
Markups won't work if notify contains something in <>

### ISSUE THREAD

**Issue by HptmHavoc** [dunst-project/dunst#213]
So I experienced that Markups won't work if the notifytext itself contains somethink with <>. For Example 
notify-send <b>Dunst</b> <is good>, output will look like; <b>Dunst</b> <is good>.  Same behaviour with & which i can't escape.

**Comment by gs93** [comment]
This is a known issue, see #99.

**Comment by wavexx** [comment]
Note that PR #210 should improve the situation. You can decide, based on rules, whether incoming text should be escaped or passed-through for Pango to interpret.

**Comment by hughdavenport** [comment]
I can confirm sending `notify-send "<><i>italics</i><b>bold</b>"`  doesn't work as expected while `notify-send "<><i>italics</i><b>bold</b>"` does.

Had quick look at PR #210, looks like it could help, and issue #99 looks a bit different but also similar so may not be a true duplicate. Will get to that when I get to that issue.

Cheers,

Hugh

**Comment by Vladimir-csp** [comment]
+1 on this. Needrestart tool sends this kind of messages:
```
<p>Your session is running obsolete binaries or libraries as listed below.</p><p><i><b>Please consider to relogin or restart the affected programs!</b></i></p>\n\n...
```
Dunst displays them literary with markup tags.

**Comment by tsipinakis** [comment]
Fixed with #279, added option to control how markup is handled `full` to parse markup, `strip` to strip it and `no` to handle everything as plain text.

`<>` is not valid markup. It either needs to be escaped to `&lt;&gt;` or markup needs to be disabled with `markup = no` either globally or in a rule.
